components/ui/TopNavigation/TopNavigation.tsx

```
@@ -0,0 +1,92 @@
+        {type !== 'none' && (
+          <button onClick={onNavigationClick} className={styles.iconButton}>
+            <Icon name={type === 'back' ? 'arrow-left' : 'close'} size={24} />
+          </button>
+        )}
+      </Left>
+      <Title>
+        <span className={styles.title}>{title}</span>
+      </Title>
+      <Right>
+        {rightIcon && (
+          <button onClick={onRightIconClick} className={styles.iconButton}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)